src/clj/clj_diff/miller.clj

```diff
@@ -218,18 +224,22 @@
   (map #(vec (cons ::sentinel %)) more))
 
 (defn order->ses
-  [a b opts]
-  (let [[a* b*] (if (> (count b) (count a)) [b a] [a b])]
-    [(ses a* b* opts) a* b*]))
+  ([a b]
+   (order->ses a b {}))
+  ([a b opts]
+   (let [[a* b*] (if (> (count b) (count a)) [b a] [a b])]
+     [(ses a* b* opts) a* b*])))
 
 (defn seq-diff
-  [opts a b]
-  (let [[a b] (vectorize a b)
-        [es a* b*] (order->ses a b opts)
-        edits (apply edits opts a* b* es)]
-    (edits->script b edits (if (= a* a) identity transpose))))
-
-(defn string-dispatch [a b & [_]]
+  ([a b]
+   (seq-diff a b {}))
+  ([a b opts]
+   (let [[a b] (vectorize a b)
+         [es a* b*] (order->ses a b opts)
+         edits (apply edits a* b* es)]
+     (edits->script b edits (if (= a* a) identity transpose)))))
+
+(defn string-dispatch [a b _]
   (when (and (string? a) (string? b)) :string))
```

Comment:
I know you accepted the suggestion in 2b2330a but I'm not sure if you read through the whole comment of https://github.com/droitfintech/clj-diff/pull/8#discussion_r955728864.

```suggestion
(defn string-dispatch
  ([a b]
   (string-dispatch a b {}))
  ([a b _]
   (when (and (string? a) (string? b)) :string)))
```

This is needed because otherwise it would be an arity issue when using the 2-arity of `edit-distance` or `longest-common-subseq`. See other comment about reverting the tests back to the 2-arity.

Reply:
done